Formula/b/bitwise.rb

```diff
@@ -3,7 +3,7 @@ class Bitwise < Formula
   homepage "https://github.com/mellowcandle/bitwise"
   url "https://github.com/mellowcandle/bitwise/releases/download/v0.50/bitwise-v0.50.tar.gz"
   sha256 "806271fa5bf31de0600315e8720004a8f529954480e991ca84a9868dc1cae97e"
-  license "GPL-3.0"
+  license "GPL-3.0-only"
```

Comment:
Not sure if that is true: https://github.com/mellowcandle/bitwise/blob/master/src/shunting-yard.c#L3-L4

Reply:
"GPL-3.0" to "GPL-3.0-only" in the formula's license specification is due to a change in the SPDX, old identifier "GPL-3.0" was ambiguous. To address this ambiguity, they introduced more specific identifiers 
https://spdx.org/licenses/